docs/kagi/getting-started/setting-default.md

```diff
@@ -50,13 +50,14 @@ A modal will appear, click the **Get Your Session Link** button to get the link
 <a name="browser_extension"></a>
 ## Option 2: Using the Kagi Browser Extension {#browser_extension}
 
-Kagi is available as an extension for all major browsers. The extension sets Kagi as the default search engine and allows you to **search with Kagi even in a private window without logging in** (this can be also set manually with the use of a [private session link](https://help.kagi.com/kagi/getting-started/setting-default.html#private_session)).
+Kagi is available as an extension for all major browsers. The extension sets Kagi as the default search engine and allows you to **search with Kagi even in a private window without logging in** (this can be also set manually with the use of a [private session link](#private_session)).
```

Comment:
Since this link (and some others I updated) refers to the same doc, this way the link works even locally, when developing.